cmd/olympus/actions/actions_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 var (
-	testConfigFile = filepath.Join("..", "..", "..", "config.test.toml")
```

Comment:
i'm thinking about keeping it test in test and .dev. as a default in a code

Reply:
@michalpristas I don't mind us deleting `config.test.toml` unless there's something about the test.toml file that is different from the regular one? We can always introduce it later on if the case turns out to be true.

Reply:
At the moment they are identical.

Reply:
if they're identical then one of them is redundant

Reply:
yep, so let's merge this and if we ever need a special config values for tests we can put this file back, but i'd argue that we can just pass those special vals directly in the tests.